src/sentry/api/validators/sentry_apps/schema.py

```diff
@@ -45,6 +45,7 @@
                 "name": {"type": "string"},
                 "uri": {"$ref": "#/definitions/uri"},
                 "options": {"$ref": "#/definitions/options"},
+                "depends": {"type": "array", "minItems": 1, "items": {"type": "string"}},
```

Comment:
@MeredithAnya I agree it's more clear but I should point out I picked `depend` because it's how we named the field for plugins (https://github.com/getsentry/sentry/blob/master/src/sentry_plugins/trello/plugin.py#L165). If you think that clarity trumps consistency here, I'll change it.

Reply:
yeah I think because this is externally facing, and users will actually be using this field, we should make it as clear as possible